addons/vehicles/vehicles/van.hpp

```diff
@@ -0,0 +1,186 @@
+class Van_02_base_F;
+class Van_02_vehicle_base_F: Van_02_base_F {
+    class EventHandlers;
+
+    class TextureSources {
+        class Verstappen {
```

Comment:
This really doesn't feel like it belongs into Theseus Services.

Reply:
No idea where else to put it tbh.

Reply:
Maybe an optional? If not GT Armory.

It's not Theseus Services that should provide it I think.

Reply:
And it's gone